boards/adafruit/qt_py_esp32s3/adafruit_qt_py_esp32s3_procpu.yaml

```diff
@@ -0,0 +1,22 @@
+identifier: adafruit_qt_py_esp32s3/esp32s3/procpu
+name: Adafruit QT Py ESP32S3 PROCPU
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+supported:
+  - gpio
+  - uart
+  - i2c
+  - spi
+  - can
+  - counter
+  - watchdog
+  - entropy
+  - pwm
+  - dma
+testing:
```

Comment:
Would you please remove the below? We have started removing this entries from all boards (another PR). Would be good not to keep adding this anymore.
```c
testing:
  ignore_tags:
    - net
    - bluetooth
```

Reply:
Should I also be removing the `net` and `bluetooth` from the `ignore_tags` section of the appcpu.yaml file?

Reply:
is it PR #80785 ?

Reply:
I took the lines out of the procpu.yaml files.